scripts/map_all_uptake_script.py

```diff
@@ -0,0 +1,82 @@
+"""Uptake of Scouting by local authority.
+
+This script produces a boundary report by local authority district, and plots
+the percentage of young people.
+
+This script has no command line options.
+"""
+
+from src.scout_data import ScoutData
+from src.boundary import Boundary
```

Comment:
I don't think this script will run? As it calls Boundary which doesn't exist anymore

Reply:
Can this file be refactored to the current API?

Reply:
Apologies, thought I'd done this